{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "bind-9.16.20-1.4 on GA media",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "These are all security issues fixed in the bind-9.16.20-1.4 package on the GA media of openSUSE Tumbleweed.",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "openSUSE-Tumbleweed-2024-10650",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/opensuse-su-2024_10650-1.json"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2006-4339 page",
				"url": "https://www.suse.com/security/cve/CVE-2006-4339/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2007-2925 page",
				"url": "https://www.suse.com/security/cve/CVE-2007-2925/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2007-2926 page",
				"url": "https://www.suse.com/security/cve/CVE-2007-2926/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2015-8461 page",
				"url": "https://www.suse.com/security/cve/CVE-2015-8461/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2016-2775 page",
				"url": "https://www.suse.com/security/cve/CVE-2016-2775/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2016-9131 page",
				"url": "https://www.suse.com/security/cve/CVE-2016-9131/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2016-9778 page",
				"url": "https://www.suse.com/security/cve/CVE-2016-9778/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2017-3135 page",
				"url": "https://www.suse.com/security/cve/CVE-2017-3135/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2017-3141 page",
				"url": "https://www.suse.com/security/cve/CVE-2017-3141/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2017-3142 page",
				"url": "https://www.suse.com/security/cve/CVE-2017-3142/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2017-3145 page",
				"url": "https://www.suse.com/security/cve/CVE-2017-3145/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2018-5737 page",
				"url": "https://www.suse.com/security/cve/CVE-2018-5737/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2018-5743 page",
				"url": "https://www.suse.com/security/cve/CVE-2018-5743/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2018-5745 page",
				"url": "https://www.suse.com/security/cve/CVE-2018-5745/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2019-6465 page",
				"url": "https://www.suse.com/security/cve/CVE-2019-6465/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2019-6471 page",
				"url": "https://www.suse.com/security/cve/CVE-2019-6471/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2019-6476 page",
				"url": "https://www.suse.com/security/cve/CVE-2019-6476/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2019-6477 page",
				"url": "https://www.suse.com/security/cve/CVE-2019-6477/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8616 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8616/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8617 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8617/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8618 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8618/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8619 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8619/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8620 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8620/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8621 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8621/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8622 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8622/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8623 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8623/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8624 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8624/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-8625 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-8625/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2021-25214 page",
				"url": "https://www.suse.com/security/cve/CVE-2021-25214/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2021-25215 page",
				"url": "https://www.suse.com/security/cve/CVE-2021-25215/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2021-25216 page",
				"url": "https://www.suse.com/security/cve/CVE-2021-25216/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2021-25218 page",
				"url": "https://www.suse.com/security/cve/CVE-2021-25218/"
			}
		],
		"title": "bind-9.16.20-1.4 on GA media",
		"tracking": {
			"current_release_date": "2024-06-15T00:00:00Z",
			"generator": {
				"date": "2024-06-15T00:00:00Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "openSUSE-SU-2024:10650-1",
			"initial_release_date": "2024-06-15T00:00:00Z",
			"revision_history": [
				{
					"date": "2024-06-15T00:00:00Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "bind-9.16.20-1.4.aarch64",
								"product": {
									"name": "bind-9.16.20-1.4.aarch64",
									"product_id": "bind-9.16.20-1.4.aarch64"
								}
							},
							{
								"category": "product_version",
								"name": "bind-doc-9.16.20-1.4.aarch64",
								"product": {
									"name": "bind-doc-9.16.20-1.4.aarch64",
									"product_id": "bind-doc-9.16.20-1.4.aarch64"
								}
							},
							{
								"category": "product_version",
								"name": "bind-utils-9.16.20-1.4.aarch64",
								"product": {
									"name": "bind-utils-9.16.20-1.4.aarch64",
									"product_id": "bind-utils-9.16.20-1.4.aarch64"
								}
							},
							{
								"category": "product_version",
								"name": "python3-bind-9.16.20-1.4.aarch64",
								"product": {
									"name": "python3-bind-9.16.20-1.4.aarch64",
									"product_id": "python3-bind-9.16.20-1.4.aarch64"
								}
							}
						],
						"category": "architecture",
						"name": "aarch64"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "bind-9.16.20-1.4.ppc64le",
								"product": {
									"name": "bind-9.16.20-1.4.ppc64le",
									"product_id": "bind-9.16.20-1.4.ppc64le"
								}
							},
							{
								"category": "product_version",
								"name": "bind-doc-9.16.20-1.4.ppc64le",
								"product": {
									"name": "bind-doc-9.16.20-1.4.ppc64le",
									"product_id": "bind-doc-9.16.20-1.4.ppc64le"
								}
							},
							{
								"category": "product_version",
								"name": "bind-utils-9.16.20-1.4.ppc64le",
								"product": {
									"name": "bind-utils-9.16.20-1.4.ppc64le",
									"product_id": "bind-utils-9.16.20-1.4.ppc64le"
								}
							},
							{
								"category": "product_version",
								"name": "python3-bind-9.16.20-1.4.ppc64le",
								"product": {
									"name": "python3-bind-9.16.20-1.4.ppc64le",
									"product_id": "python3-bind-9.16.20-1.4.ppc64le"
								}
							}
						],
						"category": "architecture",
						"name": "ppc64le"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "bind-9.16.20-1.4.s390x",
								"product": {
									"name": "bind-9.16.20-1.4.s390x",
									"product_id": "bind-9.16.20-1.4.s390x"
								}
							},
							{
								"category": "product_version",
								"name": "bind-doc-9.16.20-1.4.s390x",
								"product": {
									"name": "bind-doc-9.16.20-1.4.s390x",
									"product_id": "bind-doc-9.16.20-1.4.s390x"
								}
							},
							{
								"category": "product_version",
								"name": "bind-utils-9.16.20-1.4.s390x",
								"product": {
									"name": "bind-utils-9.16.20-1.4.s390x",
									"product_id": "bind-utils-9.16.20-1.4.s390x"
								}
							},
							{
								"category": "product_version",
								"name": "python3-bind-9.16.20-1.4.s390x",
								"product": {
									"name": "python3-bind-9.16.20-1.4.s390x",
									"product_id": "python3-bind-9.16.20-1.4.s390x"
								}
							}
						],
						"category": "architecture",
						"name": "s390x"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "bind-9.16.20-1.4.x86_64",
								"product": {
									"name": "bind-9.16.20-1.4.x86_64",
									"product_id": "bind-9.16.20-1.4.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "bind-doc-9.16.20-1.4.x86_64",
								"product": {
									"name": "bind-doc-9.16.20-1.4.x86_64",
									"product_id": "bind-doc-9.16.20-1.4.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "bind-utils-9.16.20-1.4.x86_64",
								"product": {
									"name": "bind-utils-9.16.20-1.4.x86_64",
									"product_id": "bind-utils-9.16.20-1.4.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "python3-bind-9.16.20-1.4.x86_64",
								"product": {
									"name": "python3-bind-9.16.20-1.4.x86_64",
									"product_id": "python3-bind-9.16.20-1.4.x86_64"
								}
							}
						],
						"category": "architecture",
						"name": "x86_64"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-9.16.20-1.4.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64"
				},
				"product_reference": "bind-9.16.20-1.4.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-9.16.20-1.4.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le"
				},
				"product_reference": "bind-9.16.20-1.4.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-9.16.20-1.4.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-9.16.20-1.4.s390x"
				},
				"product_reference": "bind-9.16.20-1.4.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-9.16.20-1.4.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64"
				},
				"product_reference": "bind-9.16.20-1.4.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-doc-9.16.20-1.4.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64"
				},
				"product_reference": "bind-doc-9.16.20-1.4.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-doc-9.16.20-1.4.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le"
				},
				"product_reference": "bind-doc-9.16.20-1.4.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-doc-9.16.20-1.4.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x"
				},
				"product_reference": "bind-doc-9.16.20-1.4.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-doc-9.16.20-1.4.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64"
				},
				"product_reference": "bind-doc-9.16.20-1.4.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-utils-9.16.20-1.4.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64"
				},
				"product_reference": "bind-utils-9.16.20-1.4.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-utils-9.16.20-1.4.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le"
				},
				"product_reference": "bind-utils-9.16.20-1.4.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-utils-9.16.20-1.4.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x"
				},
				"product_reference": "bind-utils-9.16.20-1.4.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "bind-utils-9.16.20-1.4.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64"
				},
				"product_reference": "bind-utils-9.16.20-1.4.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-bind-9.16.20-1.4.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64"
				},
				"product_reference": "python3-bind-9.16.20-1.4.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-bind-9.16.20-1.4.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le"
				},
				"product_reference": "python3-bind-9.16.20-1.4.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-bind-9.16.20-1.4.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x"
				},
				"product_reference": "python3-bind-9.16.20-1.4.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-bind-9.16.20-1.4.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				},
				"product_reference": "python3-bind-9.16.20-1.4.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2006-4339",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2006-4339"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "OpenSSL before 0.9.7, 0.9.7 before 0.9.7k, and 0.9.8 before 0.9.8c, when using an RSA key with exponent 3, removes PKCS-1 padding before generating a hash, which allows remote attackers to forge a PKCS #1 v1.5 signature that is signed by that RSA key and prevents OpenSSL from correctly verifying X.509 and other certificates that use PKCS #1.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2006-4339",
					"url": "https://www.suse.com/security/cve/CVE-2006-4339"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 202366 for CVE-2006-4339",
					"url": "https://bugzilla.suse.com/202366"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 203595 for CVE-2006-4339",
					"url": "https://bugzilla.suse.com/203595"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 206636 for CVE-2006-4339",
					"url": "https://bugzilla.suse.com/206636"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 207635 for CVE-2006-4339",
					"url": "https://bugzilla.suse.com/207635"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 215623 for CVE-2006-4339",
					"url": "https://bugzilla.suse.com/215623"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 218303 for CVE-2006-4339",
					"url": "https://bugzilla.suse.com/218303"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 233584 for CVE-2006-4339",
					"url": "https://bugzilla.suse.com/233584"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 564512 for CVE-2006-4339",
					"url": "https://bugzilla.suse.com/564512"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2006-4339"
		},
		{
			"cve": "CVE-2007-2925",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2007-2925"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "The default access control lists (ACL) in ISC BIND 9.4.0, 9.4.1, and 9.5.0a1 through 9.5.0a5 do not set the allow-recursion and allow-query-cache ACLs, which allows remote attackers to make recursive queries and query the cache.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2007-2925",
					"url": "https://www.suse.com/security/cve/CVE-2007-2925"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 294403 for CVE-2007-2925",
					"url": "https://bugzilla.suse.com/294403"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2007-2925"
		},
		{
			"cve": "CVE-2007-2926",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2007-2926"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "ISC BIND 9 through 9.5.0a5 uses a weak random number generator during generation of DNS query ids when answering resolver questions or sending NOTIFY messages to slave name servers, which makes it easier for remote attackers to guess the next query id and perform DNS cache poisoning.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2007-2926",
					"url": "https://www.suse.com/security/cve/CVE-2007-2926"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 294403 for CVE-2007-2926",
					"url": "https://bugzilla.suse.com/294403"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 295040 for CVE-2007-2926",
					"url": "https://bugzilla.suse.com/295040"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2007-2926"
		},
		{
			"cve": "CVE-2015-8461",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2015-8461"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Race condition in resolver.c in named in ISC BIND 9.9.8 before 9.9.8-P2 and 9.10.3 before 9.10.3-P2 allows remote attackers to cause a denial of service (INSIST assertion failure and daemon exit) via unspecified vectors.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2015-8461",
					"url": "https://www.suse.com/security/cve/CVE-2015-8461"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 958861 for CVE-2015-8461",
					"url": "https://bugzilla.suse.com/958861"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 958862 for CVE-2015-8461",
					"url": "https://bugzilla.suse.com/958862"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2015-8461"
		},
		{
			"cve": "CVE-2016-2775",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2016-2775"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "ISC BIND 9.x before 9.9.9-P2, 9.10.x before 9.10.4-P2, and 9.11.x before 9.11.0b2, when lwresd or the named lwres option is enabled, allows remote attackers to cause a denial of service (daemon crash) via a long request that uses the lightweight resolver protocol.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2016-2775",
					"url": "https://www.suse.com/security/cve/CVE-2016-2775"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 989528 for CVE-2016-2775",
					"url": "https://bugzilla.suse.com/989528"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.9,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "low"
				}
			],
			"title": "CVE-2016-2775"
		},
		{
			"cve": "CVE-2016-9131",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2016-9131"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "named in ISC BIND 9.x before 9.9.9-P5, 9.10.x before 9.10.4-P5, and 9.11.x before 9.11.0-P2 allows remote attackers to cause a denial of service (assertion failure and daemon exit) via a malformed response to an RTYPE ANY query.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2016-9131",
					"url": "https://www.suse.com/security/cve/CVE-2016-9131"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1018699 for CVE-2016-9131",
					"url": "https://bugzilla.suse.com/1018699"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1018700 for CVE-2016-9131",
					"url": "https://bugzilla.suse.com/1018700"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1018701 for CVE-2016-9131",
					"url": "https://bugzilla.suse.com/1018701"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1018702 for CVE-2016-9131",
					"url": "https://bugzilla.suse.com/1018702"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1033466 for CVE-2016-9131",
					"url": "https://bugzilla.suse.com/1033466"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2016-9131"
		},
		{
			"cve": "CVE-2016-9778",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2016-9778"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An error in handling certain queries can cause an assertion failure when a server is using the nxdomain-redirect feature to cover a zone for which it is also providing authoritative service. A vulnerable server could be intentionally stopped by an attacker if it was using a configuration that met the criteria for the vulnerability and if the attacker could cause it to accept a query that possessed the required attributes. Please note: This vulnerability affects the \"nxdomain-redirect\" feature, which is one of two methods of handling NXDOMAIN redirection, and is only available in certain versions of BIND. Redirection using zones of type \"redirect\" is not affected by this vulnerability. Affects BIND 9.9.8-S1 -> 9.9.8-S3, 9.9.9-S1 -> 9.9.9-S6, 9.11.0-9.11.0-P1.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2016-9778",
					"url": "https://www.suse.com/security/cve/CVE-2016-9778"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1018699 for CVE-2016-9778",
					"url": "https://bugzilla.suse.com/1018699"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1018703 for CVE-2016-9778",
					"url": "https://bugzilla.suse.com/1018703"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.9,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2016-9778"
		},
		{
			"cve": "CVE-2017-3135",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2017-3135"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Under some conditions when using both DNS64 and RPZ to rewrite query responses, query processing can resume in an inconsistent state leading to either an INSIST assertion failure or an attempt to read through a NULL pointer. Affects BIND 9.8.8, 9.9.3-S1 -> 9.9.9-S7, 9.9.3 -> 9.9.9-P5, 9.9.10b1, 9.10.0 -> 9.10.4-P5, 9.10.5b1, 9.11.0 -> 9.11.0-P2, 9.11.1b1.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2017-3135",
					"url": "https://www.suse.com/security/cve/CVE-2017-3135"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1018700 for CVE-2017-3135",
					"url": "https://bugzilla.suse.com/1018700"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1018701 for CVE-2017-3135",
					"url": "https://bugzilla.suse.com/1018701"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1018702 for CVE-2017-3135",
					"url": "https://bugzilla.suse.com/1018702"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1024130 for CVE-2017-3135",
					"url": "https://bugzilla.suse.com/1024130"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1033466 for CVE-2017-3135",
					"url": "https://bugzilla.suse.com/1033466"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2017-3135"
		},
		{
			"cve": "CVE-2017-3141",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2017-3141"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "The BIND installer on Windows uses an unquoted service path which can enable a local user to achieve privilege escalation if the host file system permissions allow this. Affects BIND 9.2.6-P2->9.2.9, 9.3.2-P1->9.3.6, 9.4.0->9.8.8, 9.9.0->9.9.10, 9.10.0->9.10.5, 9.11.0->9.11.1, 9.9.3-S1->9.9.10-S1, 9.10.5-S1.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2017-3141",
					"url": "https://www.suse.com/security/cve/CVE-2017-3141"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1044225 for CVE-2017-3141",
					"url": "https://bugzilla.suse.com/1044225"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1044226 for CVE-2017-3141",
					"url": "https://bugzilla.suse.com/1044226"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.8,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2017-3141"
		},
		{
			"cve": "CVE-2017-3142",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2017-3142"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An attacker who is able to send and receive messages to an authoritative DNS server and who has knowledge of a valid TSIG key name may be able to circumvent TSIG authentication of AXFR requests via a carefully constructed request packet. A server that relies solely on TSIG keys for protection with no other ACL protection could be manipulated into: providing an AXFR of a zone to an unauthorized recipient or accepting bogus NOTIFY packets. Affects BIND 9.4.0->9.8.8, 9.9.0->9.9.10-P1, 9.10.0->9.10.5-P1, 9.11.0->9.11.1-P1, 9.9.3-S1->9.9.10-S2, 9.10.5-S1->9.10.5-S2.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2017-3142",
					"url": "https://www.suse.com/security/cve/CVE-2017-3142"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1024130 for CVE-2017-3142",
					"url": "https://bugzilla.suse.com/1024130"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1046554 for CVE-2017-3142",
					"url": "https://bugzilla.suse.com/1046554"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1046555 for CVE-2017-3142",
					"url": "https://bugzilla.suse.com/1046555"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.3,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2017-3142"
		},
		{
			"cve": "CVE-2017-3145",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2017-3145"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "BIND was improperly sequencing cleanup operations on upstream recursion fetch contexts, leading in some cases to a use-after-free error that can trigger an assertion failure and crash in named. Affects BIND 9.0.0 to 9.8.x, 9.9.0 to 9.9.11, 9.10.0 to 9.10.6, 9.11.0 to 9.11.2, 9.9.3-S1 to 9.9.11-S1, 9.10.5-S1 to 9.10.6-S1, 9.12.0a1 to 9.12.0rc1.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2017-3145",
					"url": "https://www.suse.com/security/cve/CVE-2017-3145"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1076118 for CVE-2017-3145",
					"url": "https://bugzilla.suse.com/1076118"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1101131 for CVE-2017-3145",
					"url": "https://bugzilla.suse.com/1101131"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1177790 for CVE-2017-3145",
					"url": "https://bugzilla.suse.com/1177790"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2017-3145"
		},
		{
			"cve": "CVE-2018-5737",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2018-5737"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "A problem with the implementation of the new serve-stale feature in BIND 9.12 can lead to an assertion failure in rbtdb.c, even when stale-answer-enable is off. Additionally, problematic interaction between the serve-stale feature and NSEC aggressive negative caching can in some cases cause undesirable behavior from named, such as a recursion loop or excessive logging. Deliberate exploitation of this condition could cause operational problems depending on the particular manifestation -- either degradation or denial of service. Affects BIND 9.12.0 and 9.12.1.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2018-5737",
					"url": "https://www.suse.com/security/cve/CVE-2018-5737"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1093448 for CVE-2018-5737",
					"url": "https://bugzilla.suse.com/1093448"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1093449 for CVE-2018-5737",
					"url": "https://bugzilla.suse.com/1093449"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2018-5737"
		},
		{
			"cve": "CVE-2018-5743",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2018-5743"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "By design, BIND is intended to limit the number of TCP clients that can be connected at any given time. The number of allowed connections is a tunable parameter which, if unset, defaults to a conservative value for most servers. Unfortunately, the code which was intended to limit the number of simultaneous connections contained an error which could be exploited to grow the number of simultaneous connections beyond this limit. Versions affected: BIND 9.9.0 -> 9.10.8-P1, 9.11.0 -> 9.11.6, 9.12.0 -> 9.12.4, 9.14.0. BIND 9 Supported Preview Edition versions 9.9.3-S1 -> 9.11.5-S3, and 9.11.5-S5. Versions 9.13.0 -> 9.13.7 of the 9.13 development branch are also affected. Versions prior to BIND 9.9.0 have not been evaluated for vulnerability to CVE-2018-5743.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2018-5743",
					"url": "https://www.suse.com/security/cve/CVE-2018-5743"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1133185 for CVE-2018-5743",
					"url": "https://bugzilla.suse.com/1133185"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1148887 for CVE-2018-5743",
					"url": "https://bugzilla.suse.com/1148887"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1157051 for CVE-2018-5743",
					"url": "https://bugzilla.suse.com/1157051"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2018-5743"
		},
		{
			"cve": "CVE-2018-5745",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2018-5745"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "\"managed-keys\" is a feature which allows a BIND resolver to automatically maintain the keys used by trust anchors which operators configure for use in DNSSEC validation. Due to an error in the managed-keys feature it is possible for a BIND server which uses managed-keys to exit due to an assertion failure if, during key rollover, a trust anchor's keys are replaced with keys which use an unsupported algorithm. Versions affected: BIND 9.9.0 -> 9.10.8-P1, 9.11.0 -> 9.11.5-P1, 9.12.0 -> 9.12.3-P1, and versions 9.9.3-S1 -> 9.11.5-S3 of BIND 9 Supported Preview Edition. Versions 9.13.0 -> 9.13.6 of the 9.13 development branch are also affected. Versions prior to BIND 9.9.0 have not been evaluated for vulnerability to CVE-2018-5745.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2018-5745",
					"url": "https://www.suse.com/security/cve/CVE-2018-5745"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1126068 for CVE-2018-5745",
					"url": "https://bugzilla.suse.com/1126068"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1141730 for CVE-2018-5745",
					"url": "https://bugzilla.suse.com/1141730"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1148887 for CVE-2018-5745",
					"url": "https://bugzilla.suse.com/1148887"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1177790 for CVE-2018-5745",
					"url": "https://bugzilla.suse.com/1177790"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 4.9,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:H/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2018-5745"
		},
		{
			"cve": "CVE-2019-6465",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2019-6465"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Controls for zone transfers may not be properly applied to Dynamically Loadable Zones (DLZs) if the zones are writable Versions affected: BIND 9.9.0 -> 9.10.8-P1, 9.11.0 -> 9.11.5-P2, 9.12.0 -> 9.12.3-P2, and versions 9.9.3-S1 -> 9.11.5-S3 of BIND 9 Supported Preview Edition. Versions 9.13.0 -> 9.13.6 of the 9.13 development branch are also affected. Versions prior to BIND 9.9.0 have not been evaluated for vulnerability to CVE-2019-6465.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2019-6465",
					"url": "https://www.suse.com/security/cve/CVE-2019-6465"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1126069 for CVE-2019-6465",
					"url": "https://bugzilla.suse.com/1126069"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1141730 for CVE-2019-6465",
					"url": "https://bugzilla.suse.com/1141730"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1148887 for CVE-2019-6465",
					"url": "https://bugzilla.suse.com/1148887"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.3,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2019-6465"
		},
		{
			"cve": "CVE-2019-6471",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2019-6471"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "A race condition which may occur when discarding malformed packets can result in BIND exiting due to a REQUIRE assertion failure in dispatch.c. Versions affected: BIND 9.11.0 -> 9.11.7, 9.12.0 -> 9.12.4-P1, 9.14.0 -> 9.14.2. Also all releases of the BIND 9.13 development branch and version 9.15.0 of the BIND 9.15 development branch and BIND Supported Preview Edition versions 9.11.3-S1 -> 9.11.7-S1.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2019-6471",
					"url": "https://www.suse.com/security/cve/CVE-2019-6471"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1138687 for CVE-2019-6471",
					"url": "https://bugzilla.suse.com/1138687"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.9,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2019-6471"
		},
		{
			"cve": "CVE-2019-6476",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2019-6476"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "A defect in code added to support QNAME minimization can cause named to exit with an assertion failure if a forwarder returns a referral rather than resolving the query. This affects BIND versions 9.14.0 up to 9.14.6, and 9.15.0 up to 9.15.4.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2019-6476",
					"url": "https://www.suse.com/security/cve/CVE-2019-6476"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1153993 for CVE-2019-6476",
					"url": "https://bugzilla.suse.com/1153993"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1153994 for CVE-2019-6476",
					"url": "https://bugzilla.suse.com/1153994"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.9,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2019-6476"
		},
		{
			"cve": "CVE-2019-6477",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2019-6477"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "With pipelining enabled each incoming query on a TCP connection requires a similar resource allocation to a query received via UDP or via TCP without pipelining enabled. A client using a TCP-pipelined connection to a server could consume more resources than the server has been provisioned to handle. When a TCP connection with a large number of pipelined queries is closed, the load on the server releasing these multiple resources can cause it to become unresponsive, even for queries that can be answered authoritatively or from cache. (This is most likely to be perceived as an intermittent server problem).",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2019-6477",
					"url": "https://www.suse.com/security/cve/CVE-2019-6477"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1157051 for CVE-2019-6477",
					"url": "https://bugzilla.suse.com/1157051"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1197136 for CVE-2019-6477",
					"url": "https://bugzilla.suse.com/1197136"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2019-6477"
		},
		{
			"cve": "CVE-2020-8616",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8616"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "A malicious actor who intentionally exploits this lack of effective limitation on the number of fetches performed when processing referrals can, through the use of specially crafted referrals, cause a recursing server to issue a very large number of fetches in an attempt to process the referral. This has at least two potential effects: The performance of the recursing server can potentially be degraded by the additional work required to perform these fetches, and The attacker can exploit this behavior to use the recursing server as a reflector in a reflection attack with a high amplification factor.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8616",
					"url": "https://www.suse.com/security/cve/CVE-2020-8616"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1109160 for CVE-2020-8616",
					"url": "https://bugzilla.suse.com/1109160"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1171740 for CVE-2020-8616",
					"url": "https://bugzilla.suse.com/1171740"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 8.6,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2020-8616"
		},
		{
			"cve": "CVE-2020-8617",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8617"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Using a specially-crafted message, an attacker may potentially cause a BIND server to reach an inconsistent state if the attacker knows (or successfully guesses) the name of a TSIG key used by the server. Since BIND, by default, configures a local session key even on servers whose configuration does not otherwise make use of it, almost all current BIND servers are vulnerable. In releases of BIND dating from March 2018 and after, an assertion check in tsig.c detects this inconsistent state and deliberately exits. Prior to the introduction of the check the server would continue operating in an inconsistent state, with potentially harmful results.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8617",
					"url": "https://www.suse.com/security/cve/CVE-2020-8617"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1109160 for CVE-2020-8617",
					"url": "https://bugzilla.suse.com/1109160"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1171740 for CVE-2020-8617",
					"url": "https://bugzilla.suse.com/1171740"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2020-8617"
		},
		{
			"cve": "CVE-2020-8618",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8618"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An attacker who is permitted to send zone data to a server via zone transfer can exploit this to intentionally trigger the assertion failure with a specially constructed zone, denying service to clients.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8618",
					"url": "https://www.suse.com/security/cve/CVE-2020-8618"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1172958 for CVE-2020-8618",
					"url": "https://bugzilla.suse.com/1172958"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 4.9,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2020-8618"
		},
		{
			"cve": "CVE-2020-8619",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8619"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In ISC BIND9 versions BIND 9.11.14 -> 9.11.19, BIND 9.14.9 -> 9.14.12, BIND 9.16.0 -> 9.16.3, BIND Supported Preview Edition 9.11.14-S1 -> 9.11.19-S1: Unless a nameserver is providing authoritative service for one or more zones and at least one zone contains an empty non-terminal entry containing an asterisk (\"*\") character, this defect cannot be encountered. A would-be attacker who is allowed to change zone content could theoretically introduce such a record in order to exploit this condition to cause denial of service, though we consider the use of this vector unlikely because any such attack would require a significant privilege level and be easily traceable.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8619",
					"url": "https://www.suse.com/security/cve/CVE-2020-8619"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1172958 for CVE-2020-8619",
					"url": "https://bugzilla.suse.com/1172958"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 4.9,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2020-8619"
		},
		{
			"cve": "CVE-2020-8620",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8620"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In BIND 9.15.6 -> 9.16.5, 9.17.0 -> 9.17.3, An attacker who can establish a TCP connection with the server and send data on that connection can exploit this to trigger the assertion failure, causing the server to exit.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8620",
					"url": "https://www.suse.com/security/cve/CVE-2020-8620"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1175443 for CVE-2020-8620",
					"url": "https://bugzilla.suse.com/1175443"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1191120 for CVE-2020-8620",
					"url": "https://bugzilla.suse.com/1191120"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2020-8620"
		},
		{
			"cve": "CVE-2020-8621",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8621"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In BIND 9.14.0 -> 9.16.5, 9.17.0 -> 9.17.3, If a server is configured with both QNAME minimization and 'forward first' then an attacker who can send queries to it may be able to trigger the condition that will cause the server to crash. Servers that 'forward only' are not affected.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8621",
					"url": "https://www.suse.com/security/cve/CVE-2020-8621"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1175443 for CVE-2020-8621",
					"url": "https://bugzilla.suse.com/1175443"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1191120 for CVE-2020-8621",
					"url": "https://bugzilla.suse.com/1191120"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2020-8621"
		},
		{
			"cve": "CVE-2020-8622",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8622"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In BIND 9.0.0 -> 9.11.21, 9.12.0 -> 9.16.5, 9.17.0 -> 9.17.3, also affects 9.9.3-S1 -> 9.11.21-S1 of the BIND 9 Supported Preview Edition, An attacker on the network path for a TSIG-signed request, or operating the server receiving the TSIG-signed request, could send a truncated response to that request, triggering an assertion failure, causing the server to exit. Alternately, an off-path attacker would have to correctly guess when a TSIG-signed request was sent, along with other characteristics of the packet and message, and spoof a truncated response to trigger an assertion failure, causing the server to exit.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8622",
					"url": "https://www.suse.com/security/cve/CVE-2020-8622"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1175443 for CVE-2020-8622",
					"url": "https://bugzilla.suse.com/1175443"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1188888 for CVE-2020-8622",
					"url": "https://bugzilla.suse.com/1188888"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1191120 for CVE-2020-8622",
					"url": "https://bugzilla.suse.com/1191120"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 6.5,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2020-8622"
		},
		{
			"cve": "CVE-2020-8623",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8623"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In BIND 9.10.0 -> 9.11.21, 9.12.0 -> 9.16.5, 9.17.0 -> 9.17.3, also affects 9.10.5-S1 -> 9.11.21-S1 of the BIND 9 Supported Preview Edition, An attacker that can reach a vulnerable system with a specially crafted query packet can trigger a crash. To be vulnerable, the system must: * be running BIND that was built with \"--enable-native-pkcs11\" * be signing one or more zones with an RSA key * be able to receive queries from a possible attacker",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8623",
					"url": "https://www.suse.com/security/cve/CVE-2020-8623"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1175443 for CVE-2020-8623",
					"url": "https://bugzilla.suse.com/1175443"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1191120 for CVE-2020-8623",
					"url": "https://bugzilla.suse.com/1191120"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2020-8623"
		},
		{
			"cve": "CVE-2020-8624",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8624"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In BIND 9.9.12 -> 9.9.13, 9.10.7 -> 9.10.8, 9.11.3 -> 9.11.21, 9.12.1 -> 9.16.5, 9.17.0 -> 9.17.3, also affects 9.9.12-S1 -> 9.9.13-S1, 9.11.3-S1 -> 9.11.21-S1 of the BIND 9 Supported Preview Edition, An attacker who has been granted privileges to change a specific subset of the zone's content could abuse these unintended additional privileges to update other contents of the zone.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8624",
					"url": "https://www.suse.com/security/cve/CVE-2020-8624"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1175443 for CVE-2020-8624",
					"url": "https://bugzilla.suse.com/1175443"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1191120 for CVE-2020-8624",
					"url": "https://bugzilla.suse.com/1191120"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 4.3,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:L/A:N",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2020-8624"
		},
		{
			"cve": "CVE-2020-8625",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-8625"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "BIND servers are vulnerable if they are running an affected version and are configured to use GSS-TSIG features. In a configuration which uses BIND's default settings the vulnerable code path is not exposed, but a server can be rendered vulnerable by explicitly setting valid values for the tkey-gssapi-keytab or tkey-gssapi-credentialconfiguration options. Although the default configuration is not vulnerable, GSS-TSIG is frequently used in networks where BIND is integrated with Samba, as well as in mixed-server environments that combine BIND servers with Active Directory domain controllers. The most likely outcome of a successful exploitation of the vulnerability is a crash of the named process. However, remote code execution, while unproven, is theoretically possible. Affects: BIND 9.5.0 -> 9.11.27, 9.12.0 -> 9.16.11, and versions BIND 9.11.3-S1 -> 9.11.27-S1 and 9.16.8-S1 -> 9.16.11-S1 of BIND Supported Preview Edition. Also release versions 9.17.0 -> 9.17.1 of the BIND 9.17 development branch",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-8625",
					"url": "https://www.suse.com/security/cve/CVE-2020-8625"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1182246 for CVE-2020-8625",
					"url": "https://bugzilla.suse.com/1182246"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1182483 for CVE-2020-8625",
					"url": "https://bugzilla.suse.com/1182483"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1192708 for CVE-2020-8625",
					"url": "https://bugzilla.suse.com/1192708"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1196172 for CVE-2020-8625",
					"url": "https://bugzilla.suse.com/1196172"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1218478 for CVE-2020-8625",
					"url": "https://bugzilla.suse.com/1218478"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1225626 for CVE-2020-8625",
					"url": "https://bugzilla.suse.com/1225626"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 8.1,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2020-8625"
		},
		{
			"cve": "CVE-2021-25214",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2021-25214"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In BIND 9.8.5 -> 9.8.8, 9.9.3 -> 9.11.29, 9.12.0 -> 9.16.13, and versions BIND 9.9.3-S1 -> 9.11.29-S1 and 9.16.8-S1 -> 9.16.13-S1 of BIND 9 Supported Preview Edition, as well as release versions 9.17.0 -> 9.17.11 of the BIND 9.17 development branch, when a vulnerable version of named receives a malformed IXFR triggering the flaw described above, the named process will terminate due to a failed assertion the next time the transferred secondary zone is refreshed.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2021-25214",
					"url": "https://www.suse.com/security/cve/CVE-2021-25214"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1185345 for CVE-2021-25214",
					"url": "https://bugzilla.suse.com/1185345"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 6.5,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2021-25214"
		},
		{
			"cve": "CVE-2021-25215",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2021-25215"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In BIND 9.0.0 -> 9.11.29, 9.12.0 -> 9.16.13, and versions BIND 9.9.3-S1 -> 9.11.29-S1 and 9.16.8-S1 -> 9.16.13-S1 of BIND Supported Preview Edition, as well as release versions 9.17.0 -> 9.17.11 of the BIND 9.17 development branch, when a vulnerable version of named receives a query for a record triggering the flaw described above, the named process will terminate due to a failed assertion check. The vulnerability affects all currently maintained BIND 9 branches (9.11, 9.11-S, 9.16, 9.16-S, 9.17) as well as all other versions of BIND 9.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2021-25215",
					"url": "https://www.suse.com/security/cve/CVE-2021-25215"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1185345 for CVE-2021-25215",
					"url": "https://bugzilla.suse.com/1185345"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1189848 for CVE-2021-25215",
					"url": "https://bugzilla.suse.com/1189848"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1196172 for CVE-2021-25215",
					"url": "https://bugzilla.suse.com/1196172"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1199298 for CVE-2021-25215",
					"url": "https://bugzilla.suse.com/1199298"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1225626 for CVE-2021-25215",
					"url": "https://bugzilla.suse.com/1225626"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2021-25215"
		},
		{
			"cve": "CVE-2021-25216",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2021-25216"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In BIND 9.5.0 -> 9.11.29, 9.12.0 -> 9.16.13, and versions BIND 9.11.3-S1 -> 9.11.29-S1 and 9.16.8-S1 -> 9.16.13-S1 of BIND Supported Preview Edition, as well as release versions 9.17.0 -> 9.17.1 of the BIND 9.17 development branch, BIND servers are vulnerable if they are running an affected version and are configured to use GSS-TSIG features. In a configuration which uses BIND's default settings the vulnerable code path is not exposed, but a server can be rendered vulnerable by explicitly setting values for the tkey-gssapi-keytab or tkey-gssapi-credential configuration options. Although the default configuration is not vulnerable, GSS-TSIG is frequently used in networks where BIND is integrated with Samba, as well as in mixed-server environments that combine BIND servers with Active Directory domain controllers. For servers that meet these conditions, the ISC SPNEGO implementation is vulnerable to various attacks, depending on the CPU architecture for which BIND was built: For named binaries compiled for 64-bit platforms, this flaw can be used to trigger a buffer over-read, leading to a server crash. For named binaries compiled for 32-bit platforms, this flaw can be used to trigger a server crash due to a buffer overflow and possibly also to achieve remote code execution. We have determined that standard SPNEGO implementations are available in the MIT and Heimdal Kerberos libraries, which support a broad range of operating systems, rendering the ISC implementation unnecessary and obsolete. Therefore, to reduce the attack surface for BIND users, we will be removing the ISC SPNEGO implementation in the April releases of BIND 9.11 and 9.16 (it had already been dropped from BIND 9.17). We would not normally remove something from a stable ESV (Extended Support Version) of BIND, but since system libraries can replace the ISC SPNEGO implementation, we have made an exception in this case for reasons of stability and security.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2021-25216",
					"url": "https://www.suse.com/security/cve/CVE-2021-25216"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1185345 for CVE-2021-25216",
					"url": "https://bugzilla.suse.com/1185345"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1189848 for CVE-2021-25216",
					"url": "https://bugzilla.suse.com/1189848"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 8.1,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2021-25216"
		},
		{
			"cve": "CVE-2021-25218",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2021-25218"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In BIND 9.16.19, 9.17.16. Also, version 9.16.19-S1 of BIND Supported Preview Edition When a vulnerable version of named receives a query under the circumstances described above, the named process will terminate due to a failed assertion check. The vulnerability affects only BIND 9 releases 9.16.19, 9.17.16, and release 9.16.19-S1 of the BIND Supported Preview Edition.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
					"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2021-25218",
					"url": "https://www.suse.com/security/cve/CVE-2021-25218"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1189460 for CVE-2021-25218",
					"url": "https://bugzilla.suse.com/1189460"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-doc-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:bind-utils-9.16.20-1.4.x86_64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.aarch64",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.ppc64le",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.s390x",
						"openSUSE Tumbleweed:python3-bind-9.16.20-1.4.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2021-25218"
		}
	]
}
